website/src/components/common/ActiveFilters.tsx

```diff
@@ -19,7 +19,11 @@ export const ActiveFilters: FC<ActiveFiltersProps> = ({ sequenceFilter, removeFi
                     className='border-primary-600 rounded-sm border border-l-primary-600 bg-gray-100 border-l-8 pl-3 py-1 text-sm flex flex-row'
                 >
                     <span className='text-primary-900 font-light pr-1'>{label}:</span>
-                    <span className='text-primary-900 font-semibold'>{value}</span>
+                    {value === '' ? (
+                        <span className='text-primary-900 italic'>unset</span>
```

Comment:
(I'm not sure whether unset is a very intuitive term, but I also don't know what's better, so I'm happy with leaving it as is but just bringing it up in case someone else has a better idea.)

Reply:
I also considered _(none)_

Reply:
I think unset is better than none. `All` could work. But I don't think it matters too much as it's moderately niche.

Reply:
Or "any"

Reply:
I think I like "any"!